.github/workflows/beekeeper.yml

```diff
@@ -65,6 +65,9 @@ jobs:
           echo -e "127.0.0.10\tregistry.localhost" | sudo tee -a /etc/hosts
           for ((i=0; i<REPLICA; i++)); do echo -e "127.0.1.$((i+1))\tbee-${i}.localhost bee-${i}-debug.localhost"; done | sudo tee -a /etc/hosts
           timeout 30m ./beeinfra.sh install --local -r "${REPLICA}" --bootnode /dnsaddr/localhost --geth --k3s --pay-threshold 1000000000000 --postage
+      - name: Default Batch ID
+        run: |
+          echo BATCH_ID=$(curl -XPOST bee-0.localhost/stamps/1/16 | jq -r .batchID) >> $GITHUB_ENV
```

Comment:
The LHS of `BATCH_ID` should be quoted, for more see: https://github.com/koalaman/shellcheck/wiki/SC2046